### PR TITLE
Allow ES6 syntax in back-end

### PR DESCRIPTION
**Pull request by belyas**
[Trello task](https://trello.com/c/B6azIwnR/156-allow-es6-syntax-in-back-end)

I implemented this new ES6 modules system on some files to prove it's working.

You should install the new babel pacakages in `package.json` file in order to work.

### CODE REVIEW

**Comment by ArnasDickus** [comment]
Hello, @belyas your fork is outdated. Try to update it to the newest changes.
On dev branch run: 
1) git fetch upstream
2) git merge upstream/dev
3) git push origin dev

**Comment by belyas** [comment]
@ArnasDickus it says `Everything up-to-date` I did all those steps to create this PR, weird

**Comment by belyas** [comment]
@ArnasDickus I ses that my fork is up-to-date, it has your latest changes `commit f9e2d2e4f7e4391b9015499aca57c4a365a0b32d (upstream/dev)`

**Comment by ArnasDickus** [comment]
@belyas Your branch is not up to date. The problem here is that you have this line
`"test": "echo \"Error: no test specified\" && exit 1",` in package.json. 
That's why it fails.
[Your PR package.json file](https://github.com/belyas/mappypals_backend/blob/dev/package.json)
[mappypals back-end Dev PR package.json file](https://github.com/zero-to-mastery/mappypals_backend/blob/dev/package.json)

**Comment by ArnasDickus** [comment]
Since this task is light on the back-end, I review it.

File: server.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 // Should be server.js but since we are not allowed to delete other people's code, a new file.
```
Comment (ArnasDickus) [inline server.js]: Could you remove this comment? It's outdated

**Review by ArnasDickus — COMMENTED** [review]
Looks solid You just need to remove one comment in server.js and I could approve it.

File: server.js (hunk 1 above)
Comment (belyas) [reply server.js]: Done @ArnasDickus

**Pull request merged**